code/game/objects/items/weapons/storage/backpack.dm

```diff
@@ -85,6 +85,22 @@
 	var/dist = max((current_size - 2), 1)
 	explosion(loc, dist, (dist * 2), (dist * 4))
 
+/obj/item/storage/backpack/holding/huge_items
+	name = "Bag of Holding (large things)"
+	desc = "A backpack that opens into a localized pocket of Blue Space. This variant can hold huge items, at the cost of capacity"
+	icon_state = "holdingpacklarge"
+	item_state = "holdingpacklarge"
+	max_combined_w_class = 12
```

Comment:
```suggestion
	max_combined_w_class = 18
```
This was not meant to be 12, you should be able to store 2 huge items with room to spare.